portal/templates/_partials/primary-navigation.njk

```diff
@@ -74,7 +74,11 @@
   {% endif %}
 
   {% if isPaymentReportOfficer %}
-    {% set navItems = (navItems.push(reportGefUtilisationAndFeesNavItem, previousGefReportsNavItem, recordCorrectionLogNavItem), navItems) %}
+    {% if FF_FEE_RECORD_CORRECTION_ENABLED == "true" %}
+      {% set navItems = (navItems.push(reportGefUtilisationAndFeesNavItem, previousGefReportsNavItem, recordCorrectionLogNavItem), navItems) %}
+    {% else %}
+      {% set navItems = (navItems.push(reportGefUtilisationAndFeesNavItem, previousGefReportsNavItem), navItems) %}
+    {% endif %}
```

Comment:
I think it would be nicer to push the ones that are not behind the feature flag just once and not have an else:
```suggestion
    {% set navItems = (navItems.push(reportGefUtilisationAndFeesNavItem, previousGefReportsNavItem), navItems) 
    {% if FF_FEE_RECORD_CORRECTION_ENABLED == "true" %}
      {% set navItems = (navItems.push(recordCorrectionLogNavItem), navItems) %}
    {% endif %}
    %}
```